package.json

```diff
@@ -4,28 +4,48 @@
   "description": "Demo app for React Native Elements UI Library",
   "author": "Monte Thakkar",
   "private": true,
-  "main": "node_modules/expo/AppEntry.js",
+  "main": "src/index.native.js",
   "dependencies": {
+    "@babel/polyfill": "^7.2.5",
     "@expo/vector-icons": "^9.0.0",
     "expo": "^32.0.0",
+    "expo-linear-gradient": "^3.0.0",
     "lodash": "^4.17.4",
+    "prop-types": "^15.7.2",
     "react": "16.5.0",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-31.0.0.tar.gz",
+    "react-art": "^16.8.3",
+    "react-dom": "^16.8.3",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz",
     "react-native-elements": "^1.1.0",
-    "react-native-ratings": "^5.3.0",
+    "react-native-ratings": "^6.3.0",
     "react-native-touchable-scale": "2.0.0",
-    "react-navigation": "1.5.8"
+    "react-native-vector-icons": "^6.3.0",
+    "react-native-web": "^0.10.0",
+    "react-navigation": "1.5.8",
+    "react-scripts": "^2.1.5"
   },
   "scripts": {
-    "start": "expo start",
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "eject-web": "react-scripts eject",
+    "expo-start": "expo start",
     "android": "expo start --android",
     "ios": "expo start --ios",
     "eject": "expo eject",
     "prettify": "prettier --write 'src/**/*.js'",
     "clean-install": "rm -rf node_modules && npm cache clean --force && watchman watch-del-all && yarn"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.3.4",
     "babel-preset-expo": "^5.0.0",
-    "prettier": "^1.15.3"
-  }
+    "customize-cra": "^0.2.12",
+    "prettier": "^1.15.3",
+    "react-app-rewired": "^2.1.0"
```

Comment:
what do you use `customize-cra` & `react-app-rewired` for? never heard/used those before but they sound cool

Reply:
It's for `eject` escaping. See [my comment](https://github.com/react-native-training/react-native-elements/issues/1808#issuecomment-468869382)